packages/wonder-stuff-render-server/src/requests-from-cache.ts

```diff
@@ -57,6 +57,31 @@ export const asUncachedRequest = (
     return superagentRequest as any;
 };
 
+// NOTE(kevinb): This is necessary to workaround the followoing error when running
+// `yarn build:types`:
+// TS4078: Parameter 'response' of exported function has or is using private name 'Response'.
+type SAResponse = Response;
+
+// TODO(kevinb): Figure out how to move this into its own .d.ts file and have it
+// work with both `yarn typecheck` and `yarn build:types`.
+declare module "superagent" {
+    interface SuperAgentRequest {
+        cacheWhenEmpty(cacheWhenEmpty: boolean): this;
+        doQuery(doQuery: boolean): this;
+        expiration(expiration?: number): this;
+        forceUpdate(forceUpdate?: boolean): this;
+        pruneQuery(pruneQuery: Array<string>): this;
+        pruneHeader(pruneHeader: Array<string>): this;
+        prune(
+            prune: (
+                response: SAResponse,
+                gutResponse: (response: SAResponse) => any,
+            ) => any,
+        ): this;
+        responseProp(responseProp: string): this;
+    }
+}
```

Comment:
In Perseus we have .d.ts files in a types/ directory at the base.  That didn't work in this situation.  It must have something to do with the fact that this is extending a module's type whereas those are starting from scratch.  Another weird thing is that when this file was in a separate .d.ts file it worked with tsconfig-build.json but not tsconfig.json.  I can't explain that either. 😞